README.md

```diff
@@ -1,54 +1,106 @@
+  deliverable templates, or just share some relevant information about the assignment.
+* File submissions - outcomes are uploaded to the server, to facilitate sharing them between teammembers and for future
+  grading.
+* Peer feedback - workgroup members can provide anonymous feedback to each other.
+* Grading - activities are either peer-graded or staff-graded:
+    * Staff grading - staff members with appropriate roles are asked to provide grade for the workgroup.
+    * Peer grading - students are asked to evaluate other workgroups' work.
+    * Staff-grading fallback - if some of the students fail to provide grades to other workgroup, staff members can 
+      interfere and provide missing grades.
+* Omnipresent features - Group Project Navigator is always displayed and provides quick access to most commonly 
+  used features:
+    * Project navigation - jump to any stage in any activity in one click.
+    * Resources panel - all the resources in one place.
+    * Submissions panel - upload, view and change deliverables as you go.
+    * Private discussions - connect with teammates.
+    * Ask Teaching Assistant - ask course staff for help in seconds.
 
-    "XBLOCK_SETTINGS": {
-      "group_project_v2": {
-        "dashboard_details_url": "/admin/workgroup/dashboard/programs/{program_id}/courses/{course_id}/projects/{project_id}/details?activate_block_id={activity_id}",
-        "ta_review_url": "/courses/{course_id}/group_work/{group_id}?activate_block_id={activate_block_id}",
-        "access_dashboard_for_all_orgs_groups": ["super_admin"],
-        "access_dashboard_groups": ["client_admin"],
-        "ta_roles": ["assistant"]
-      }
-    },
+## Setup
+
+In order to use Group Project XBlock v2 in a course, XBlock must be installed into LMS environment and enabled in 
+course's advanced settings. Also, Group Project XBlock uses a number of instance-specific configuration variables, so 
+setting them is considered a part of setup process.
+
+### Installing XBlock into LMS
+
+If you're using edx-solutions/edx-platform fork, Group Project v2 XBlock is already installed. Otherwise, installing 
+Group Project XBlock v2 is done the very same way other XBlocks are installed:
+
+1. Open terminal (or shell) to the LMS instance.
+2. Switch to LMS virtualenv
+3. pip install -e git+https://github.com/open-craft/xblock-group-project-v2.git@ **version_hash** #egg=xblock-group-project-v2
+
+*version_hash* is the hash of the git commit. By the time of writing, latest stable version was `4322ca8092c5385d5602143077e388017fcb1249`,
+so the above command looked like this:
+
+    pip install -e git+https://github.com/open-craft/xblock-group-project-v2.git@4322ca8092c5385d5602143077e388017fcb1249#egg=xblock-group-project-v2
+    
+### Setting configuration variables
+
+There are two sources of configuration variables: django settings and XBlock-specific settings available 
```

Comment:
@e-kolpakov Nit: "django" ⟶ "Django" (there's a few more instances of this below)